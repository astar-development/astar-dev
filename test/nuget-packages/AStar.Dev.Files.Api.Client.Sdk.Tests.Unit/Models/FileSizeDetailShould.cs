using AStar.Dev.Files.Api.Client.SDK.Models;
using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

[TestSubject(typeof(FileSizeDetail))]
public class FileSizeDetailShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new FileSizeDetail(123456, 789, 987).ToJson().ShouldMatchApproved();
}
