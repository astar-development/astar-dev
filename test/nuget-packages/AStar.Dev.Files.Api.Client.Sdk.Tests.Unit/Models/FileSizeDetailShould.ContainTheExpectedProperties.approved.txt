{
  "fileSize": 123456,
  "height": 789,
  "width": 987,
  "fileSizeForDisplay": "120.56 Kb"
}